src/compiler/diagnosticMessages.json

```diff
@@ -879,7 +879,6 @@
         "category": "Error",
         "code": 1271
     },
-

```

Comment:
as Nathan commented, please leave the blank lines as-is

Reply:
- [x] @gabritto Thanks! I've restored the blank lines.